### PR TITLE
Added list element

### PR DESCRIPTION
**Pull request by larnedo**
Please review @nguasch @PablitoGS.

(Pending validation with typography)

### CODE REVIEW

File: src/base/_lists.scss [hunk 1]
```diff
@@ -0,0 +1,175 @@
+// Dependencies from other packages --------------------------------------------
+@import '~sui-settings/src/settings';
+@import '../vars';
+
+@if $is-theme == false {
+  @import '~sui-typography/src/typography';
+}
+
```
Comment (PablitoGS) [inline src/base/_lists.scss]: These imports are unnecessary, they are in base.scss. @larnedo

Comment (larnedo) [inline src/base/_lists.scss]: Thanks @PablitoGS. I fix this!

**Comment by nguasch** [comment]
:+1:

File: src/base/_lists.scss [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+Styleguide Lists-1
+*/
+
+/*
+List
+
+Use class `.List` for a basic list without decoration.
+
+Markup:
+<!-- Ordered list -->
```
Comment (PablitoGS) [inline src/base/_lists.scss]: unordered and ordered list @larnedo

Comment (larnedo) [inline src/base/_lists.scss]: You're right!

**Comment by PablitoGS** [comment]
:+1:

**Pull request merged**